template/deployments/helm/draft.yaml

```diff
@@ -23,3 +23,5 @@ variableDefaults:
   - name: "IMAGETAG"
     value: "latest"
     disablePrompt: true
+  - name: "APPNAME"
```

Comment:
why is this addition needed?

Reply:
without a default appname, we can't add a test case for taking every default to validate that they work, but it looks like the GH runners don't support the `yes` command anyway, so i may remove the test

Reply:
good point i'll remove this and defer to a later PR